ReactAndroid/gradle.properties

```diff
@@ -12,7 +12,7 @@ JUNIT_VERSION=4.12
 FEST_ASSERT_CORE_VERSION=2.0M10
 
 ANDROID_SUPPORT_TEST_VERSION=1.0.2
-FRESCO_VERSION=1.10.0
```

Comment:
RN 59 uses fresco 1.10.0 
https://github.com/facebook/react-native/blob/v0.59.0/ReactAndroid/gradle.properties

Reply:
Yes.. you are right. I will revert these changes.